### PR TITLE
[workflow] Add workflow to automatically bump the s390x runner version

### PR DESCRIPTION
**Pull request by chantra**
Get the last release from GH API and update the relevant line in the Dockerfile

### CODE REVIEW

**Review by danielocfb — COMMENTED** [review]
I suppose that approach-wise this is probably fine, but I am a little hesitant about having some workflow just bump a version without anybody noticing. Not that it is necessarily much different right now, but it's also not great right now :-)
Is there a way for us to create a pull request instead? (Edit: I inferred that the action in question just pushes; if it already creates a pull request then that's fine, but that was not my reading of the docs)

**Comment by danielocfb** [comment]
Also, just curious, but have you tested what happens when the version stays the same? Does the action just not create a commit and do nothing?

**Comment by chantra** [comment]
> Also, just curious, but have you tested what happens when the version stays the same? Does the action just not create a commit and do nothing?

that's what the action documentation says, yes. https://github.com/marketplace/actions/git-auto-commit

**Comment by chantra** [comment]
> I suppose that approach-wise this is probably fine, but I am a little hesitant about having some workflow just bump a version without anybody noticing. Not that it is necessarily much different right now, but it's also not great right now :-) Is there a way for us to create a pull request instead? (Edit: I inferred that the action in question just pushes; if it already creates a pull request then that's fine, but that was not my reading of the docs)

yes, currently, at least for the non s390x, upstream eventualy update the version and it trickles down to us.
I don't think the current action I am using can create a PR, but maybe another one, I would have to search for it.

**Comment by chantra** [comment]
Ok, I found [peter-evans/create-pull-request@v4](https://github.com/peter-evans/create-pull-request) which is similar and just create PRs.

Tested on my repo.

PR created: https://github.com/chantra/runner/pull/15
actual run:
https://github.com/chantra/runner/actions/runs/4011625579

**Comment by chantra** [comment]
I am less sure if the team-reviewers feature will work or not, let see if on next run that creates a PR.

File: .github/workflows/version_bump.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: s390x version bump
+
+# This workflow pulls the latest official actions runner version using GH API
+# and update our s390x Dockerfile.
+# If a change happens, the chnage will be commited.
```
Comment (danielocfb) [inline .github/workflows/version_bump.yml:5]: Typo; should this sentence have to be updated?

**Review by danielocfb — APPROVED** [review]
Looks good, thanks!

**Pull request merged**